lib/newgistics/response_handlers/search.rb

```diff
@@ -29,6 +29,10 @@ def handle_successful_response(response)
         end
       end
 
+      def error_nodes(xml)
+        xml.css('errors error') + xml.css('Errors Error')
```

Comment:
With this change we'll also stop eating errors silently, like the security error that was being raised when querying for too many shipments